packages/plugins/plugin-mixpanel/src/methods/__tests__/identify.test.ts

```diff
@@ -0,0 +1,186 @@
+import identify from '../identify';
+import type {
+  SegmentMixpanelSettings,
+  IdentifyEventType,
+} from '@segment/analytics-react-native';
+import { mockIntegrationSettings } from '../__mocks__/mockIntegrationSettings';
+import { Mixpanel } from '../__mocks__/mixpanel-react-native';
+
+describe('#identify', () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+  it('calls identify with userId', () => {
+    const payload = {
+      type: 'identify',
+      traits: {
+        firstName: 'John',
+        phone: '(555) 555-5555',
+        foo: 'bar',
+      },
+      userId: 'user',
+    } as IdentifyEventType;
+    const settings: SegmentMixpanelSettings =
+      mockIntegrationSettings.integrations.Mixpanel;
+    const mixpanel = new Mixpanel('1234');
+
+    identify(payload, mixpanel, settings);
+
+    expect(mixpanel.identify).toBeCalled();
```

Comment:
For all these tests I think we should confirm that we are calling these methods in mixpanel with the right values, not only that the calls are done.
If the values are too complex to write here make use of snapshots: https://jestjs.io/docs/snapshot-testing